### PR TITLE
Change "up-to-date" to "up to date" where necessary

### PR DESCRIPTION
**Pull request by DePasqualeOrg**
The correct usage is _[noun] is up to date_ and _an up-to-date [noun]_. That is, hyphens are only used when _up-to-date_ is used as an adjective preceding a noun. Two examples from the New Oxford American Dictionary for reference:

> a modern, up-to-date hospital

> the book will keep you up to date

### CODE REVIEW

**Review by zkochan — APPROVED** [review]

File: packages/resolve-dependencies/CHANGELOG.md [hunk 1]
```diff
@@ -1391,7 +1391,7 @@
 
 ### Patch Changes
 
-- e2c4fdad5: Don't remove resolved peer dependencies from dependencies when lockfile is partially up-to-date.
+- e2c4fdad5: Don't remove resolved peer dependencies from dependencies when lockfile is partially up to date.
```
Comment (BlackHole1) [inline packages/resolve-dependencies/CHANGELOG.md]: This should not be changed

File: packages/pnpm/CHANGELOG.md [hunk 2]
```diff
@@ -2620,7 +2620,7 @@
 
   - The current branch is your publish branch. The publish branch is `master` by default. This is configurable through the `publish-branch` setting.
   - Your working directory is clean (there are no uncommitted changes).
-  - The branch is up-to-date.
+  - The branch is up to date.
```
Comment (BlackHole1) [inline packages/pnpm/CHANGELOG.md]: This should not be changed

**Review by BlackHole1 — COMMENTED** [review]
The contents of CHANGLOG should not be changed

**Review by BlackHole1 — APPROVED** [review]
LGTM

**Comment by DePasqualeOrg** [comment]
Thanks, I've removed the CHANGELOG files from the commit.

**Pull request merged**